test/client/fixtures/sandbox/iframe-test.js

```diff
@@ -43,6 +43,24 @@ test('event should not raise if a cross-domain iframe is appended', function ()
     $iframe.remove();
 });
 
+asyncTest('the AMD module loader damages proxing an iframe without src', function () {
```

Comment:
It should be in `regression` section and test title should inlclude github issue number (in format `(GH-{num})`)